package.json

```diff
@@ -15,8 +15,9 @@
     "dist:win": "yarn clean:dist && yarn syncversion && yarn compile && cross-env CSC_IDENTITY_AUTO_DISCOVERY=false electron-builder --win --x64",
     "pack": "yarn dist --dir -c.compression=store -c.mac.identity=null",
     "lint": "eslint ./src/**/*.ts ./src/**/*.tsx --fix",
-    "update:whistle": "rm -rf whistle && git clone https://github.com/avwo/whistle --depth=1 && patch -u -b whistle/lib/inspectors/res.js -i whistle.patch",
-    "update:node_modules": "npx rimraf files/node/modules && cd files/node && echo {} > package.json && yarn add ../../whistle-start ../../whistle whistle.vase whistle.scriptfile && mv node_modules modules",
+    "update:whistle": "echo skip",
```

Comment:
If `update:whistle` script doesn't need anymore, we can be deleted?

Reply:
yes we can😆

Reply:
fixed with d598dc9